Content.Shared/Teleportation/Systems/SharedPortalSystem.cs

```diff
@@ -46,8 +47,8 @@ public override void Initialize()
 
     private void OnGetVerbs(EntityUid uid, PortalComponent component, GetVerbsEvent<AlternativeVerb> args)
     {
-        // Traversal altverb for ghosts to use that bypasses normal functionality
-        if (!args.CanAccess || !HasComp<GhostComponent>(args.User))
+        // Traversal altverb for ghosts and revenants to use that bypasses normal functionality
+        if (!args.CanAccess || !(HasComp<GhostComponent>(args.User) || HasComp<RevenantComponent>(args.User)))
             return;
```

Comment:
Why doesn't this just use collision?

Reply:
I didn't think of doing collision as this was the simplest solution
Also I thought of a oversight so either way I need to do collision instead

Reply:
What would be your preferred way to implement this? I'm working with TheFlyingSentry on this issue and since we're both new to SS14's codebase we need some pointers.
Is there a file that defines what can and cannot pass through teleporters?

Reply:
Just use collision layers for it